changelog.asc

```diff
@@ -2,6 +2,132 @@
+** Translations: add Uyghur language
+** Translations: https://crowdin.com/project/ankidroid/activity-stream[Incredible amount of general translation work], thank you!
+** Previewer: Added text for Cards with Empty Fronts
+** Previewer: Keyboard/controller remappable bindings
+** Previewer: support non-combining (nc:) typed answers
+** Browser: New options: default search text and ignore accents
+** Browser: Multi-column support and 'Manage columns' screen
+** Browser: Support FSRS columns (Difficulty/Stability/Retrievability)
+** Browser: Support sorting by deck name
+** Maintenance: add licenses wiki page link
+** UI: Android predictive back gesture support
+** Widgets: Better resizing support & add rounded corners
+** Reposition Field: pre-fill previous value
+** Card Template Editor: Support 'restore to default'
+** Congrats Screen: support unburying only siblings/manually buried cards
+** Reposition: Implement all upstream functionality (steps, randomize order, shift cards)
+** Reposition: Display queue positions
+** JS API: Added function to setTags and getTags
+** Streamline Debug Info
+** Error dialog: copy stack trace & debug info
+** Made more actions undoable (example, "empty filtered deck")
+* Note Editor
+** Add tooltip in note editor
+** Allow pasting images as PNG
+** Add mathjax block and mathjax chemistry
+** Fix: multimedia options opening more than once
+** Fix: prevent PNG,WEBP to JPG conversion on image crop
+** Fix: Image Occlusion note note creation with special chars in filename
+** Improve LocaleSelectionDialog
+** Check Media dialog: Anki compatibility (tag missing, media report, keep notes, make undoable)
+* Deck Options
+** Improve optimization progress notifiations
+** Fix: DeckOption become closable if it's not loaded
+* Custom Study
+** Replicate desktop custom study by tags functionality
+** improvement: disable unusable items
+** improvement: only show 'extend new/reviews' when available
+* Import
+** Improve import progress notifications
+** Fix import speed when importing from outside Anki (100x improvement?)
+** Add drag-and-drop support for txt/csv/tsv
+** Add support for sharing .txt files to AnkiDroid
+** fix opening text files from other apps (Gmail etc...)
+* Design
+** Use proper color for AlertDialogs background color in black theme
+** Improve Activity Transition Animations
+** Various improvements when using a large font
+** Right to left language support improvements
+* Preferences
+** Option to remove Deck Picker background
+** Controls: Show keyboard shortcuts if keyboard attached
+** Controls: Support previewer shortcuts
+** Controls: Support one-sided gestures
+** Controls: Improved UI for control and gesture entry
+* Shared Deck Download
+** Warn if no internet connection
+** Add 'manual download' link on download failure
+* Reviewer
+** Use https://fonts.google.com/noto/specimen/Noto+Sans+Mono/license[Noto Sans Mono font] for type-in-answer
```

Comment:
```suggestion
** Much better multilingual font support, using https://fonts.google.com/noto/specimen/Noto+Sans+Mono/license[Noto Sans Mono font] for type-in-answer
```

Reply:
I forgot that was the reason! Yes that is big